In [9]:
import pandas as pd

sample = pd.read_csv('./내국인 생활인구.csv',encoding='cp949')
sample

일자(stdr_de_id)  시간(tmzon_pd_se)  ...  여자80세이상(female_above_80_lvpop_co)      등록일자(ldadng_dt)
0          20170226                4  ...                             9.7668  2021-08-17 16:06:44
1          20180107               22  ...                            18.1781  2021-08-17 16:06:44
2          20180317                6  ...                             4.4772  2021-08-17 16:06:44
3          20190222                7  ...                             7.7510  2021-08-17 16:06:44
4          20210706                6  ...                            95.0249  2021-08-17 16:06:44
..              ...              ...  ...                                ...                  ...
495        20170603                7  ...                            17.3543  2021-08-17 16:06:44
496        20211007                5  ...                             6.1426  2021-08-17 16:06:44
497        20200819               20  ...                            13.5500  2021-08-17 16:06:44
498        20171011               11  ...                             0.0000  2021-08-17 16:06:44
499        20200725               21  ...                             4.8362  2021-08-17 16:06:44

[500 rows x 38 columns]

In [15]:
sample.head(6)
# 위에서부터 몇까지 row를 가져올지 ; default:5
sample.tail(6)
# 아래서부터 몇까지 row를 가져올지 ; default:5

일자(stdr_de_id)  시간(tmzon_pd_se)  ...  여자80세이상(female_above_80_lvpop_co)      등록일자(ldadng_dt)
494        20190130               12  ...                             3.5944  2021-08-17 16:06:44
495        20170603                7  ...                            17.3543  2021-08-17 16:06:44
496        20211007                5  ...                             6.1426  2021-08-17 16:06:44
497        20200819               20  ...                            13.5500  2021-08-17 16:06:44
498        20171011               11  ...                             0.0000  2021-08-17 16:06:44
499        20200725               21  ...                             4.8362  2021-08-17 16:06:44

[6 rows x 38 columns]

## index
index를 데이터 검색, 정렬에 사용할 수 있음.

현재 데이터는 0~499까지를 인덱스로 사용하고 있음.
일자(stdr_de_id)라는 고유값을 인덱스로 새로 지정해서 data sorting 해보자

In [17]:
sample.index

RangeIndex(start=0, stop=500, step=1)

In [33]:
indexingSample = pd.read_csv('./내국인 생활인구.csv',encoding='cp949',index_col=0) # 첫번째 컬럼을 인덱스로 지정
indexingSample.head(10)

시간(tmzon_pd_se)  행정동코드(adstrd_code_se)  ...  여자80세이상(female_above_80_lvpop_co)      등록일자(ldadng_dt)
일자(stdr_de_id)                                          ...                                                        
20170226                      4               11320515  ...                             9.7668  2021-08-17 16:06:44
20180107                     22               11380510  ...                            18.1781  2021-08-17 16:06:44
20180317                      6               11680610  ...                             4.4772  2021-08-17 16:06:44
20190222                      7               11305575  ...                             7.7510  2021-08-17 16:06:44
20210706                      6               11740660  ...                            95.0249  2021-08-17 16:06:44
20170801                      1               11530750  ...                             5.1927  2021-08-17 16:06:44
20190502                     10               11620735  ...                            21.4579  2021-08-17 16:06:44
20170408                      8               11740685  ...                             0.0104  2021-08-17 16:06:44
20200614                      1               11260655  ...                             4.1581  2021-08-17 16:06:44
20201222                     22               11350611  ...                             6.3707  2021-08-17 16:06:44

[10 rows x 37 columns]

In [37]:
indexingSample.index

Index([20170226, 20180107, 20180317, 20190222, 20210706, 20170801, 20190502,
       20170408, 20200614, 20201222,
       ...
       20190817, 20190426, 20170214, 20191123, 20190130, 20170603, 20211007,
       20200819, 20171011, 20200725],
      dtype='int64', name='일자(stdr_de_id)', length=500)